nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,299 @@
+.. _nrf_modem_dect_phy:
+
+DECT NR+ physical layer
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DECT NR+ physical layer (PHY) interface in the Modem library is used to control the DECT NR+ PHY in the DECT NR+ modem firmware.
+
+The DECT NR+ modem firmware is a variant of the nRF9161 modem firmware with a different feature set than the cellular modem firmware.
+In particular, the DECT NR+ modem firmware does not support cellular operation or the Global Navigation Satellite System (GNSS).
+
+For details about the key hardware capabilities of the nRF9161 and its DECT NR+ PHY modem firmware implementation, with regards to the DECT NR+ standard, refer to the `nRF9161 DECT NR+ product specification`_.
+
+.. note::
+   To obtain the DECT NR+ modem firmware, you must contact the Nordic Semiconductor sales department.
+
+For more details about which Modem library features are available for which modem firmware, see :ref:`nrf_modem_flavour`.
+
+Regulations
+***********
+
+DECT NR+ operates on free but regulated radio channels. The availability of the channels and the exact regulation to use them varies in different countries.
+
+.. note::
+   It is your responsibility to operate the devices according to the local regulation, in all situations.
+
+.. important::
```

Comment:
What is the criteria to decide whether something's a `warning` or `important`? I use `warning` because it's not important to use the right frequency, it is actually dangerous not to use the right frequency, as it may interfere with other equipment (in range of several hundreds of meter).

Reply:
@lemrey https://nordicsemi.atlassian.net/wiki/spaces/TECHDOC/pages/120306494/Style+and+usage+guide there is a section on that in this page.